resource-metadata/hypothesis.md

```diff
@@ -0,0 +1,139 @@
+# Resource Metadata Model
+
+## Background
+
+*This section is non-normative*
+
+This document introduces a mechanism for linking to metadata about resources in the Solid Ecosystem using HTTP Link headers. Examples of this linking mechanism include:
+
+- A binary JPEG image with a Link header, pointing to a description that has an RDF representation.
+- An LDP container with a Link header, pointing to an ACL resource that governs access controls for the contained resources.
```

Comment:
I'd suggest not to mention LDP container but just go with "container" because the Solid container (`/`) is not strictly defined as an LDP container. So, just `s/An LDP/A` would suffice.

Reply:
Removed LDP